.travis.yml

```diff
@@ -45,7 +46,7 @@ install:
   - conda update -q conda conda-build
   - conda config --add channels pcds-tag
   - conda config --add channels $PCDS_CHANNEL
-  - conda config --append channels conda-forge
+  - conda config --add channels conda-forge
```

Comment:
Why is this necessary? I'm kind of worried about reprecussions

Reply:
technically not necessary in this PR, but it was the only difference between my local config that created the correct environment and the travis config here that created the wrong environment

Reply:
Huh ok, conda is weird :(

Reply:
The solver needs to make some sort of default judgement if it has multiple mutually exclusive solutions for the environment. Since the default judgement can depend on channel order and priority, it can get messy. We should avoid less than and equal to pins at all costs in our dependencies to minimize these cases of multiple incompatible solutions.